ambry-api/src/main/java/com.github.ambry/config/RouterConfig.java

```diff
@@ -100,7 +100,7 @@
    * The maximum number of parallel requests allowed for a delete operation.
    */
   @Config("router.delete.request.parallelism")
-  @Default("12")
+  @Default("3")
```

Comment:
@xiahome this is to ensure that deletes go cross colo only if it fails locally.
@vgkholla changed the parallelism in the `DeleteManagerTest` from 9 to 3. With that change, existing tests like `testBlobNotFoundWithTwoBlobDeleted` exercise the case where local fails and the operation goes cross colo.